python/zfpy.pyx

```diff
@@ -329,15 +330,15 @@ cpdef np.ndarray _decompress(
     return output
 
 cpdef np.ndarray decompress_numpy(
-    bytes compressed_data,
+     const uint8_t[::1] compressed_data,
```

Comment:
Sorry there may have been an extra space in the suggestion before. Maybe that helps?

```suggestion
    const uint8_t[::1] compressed_data,
```